bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/driver/Port.java

```diff
@@ -161,10 +161,10 @@ public void start() {
             return;
         }
         m_readThread = new Thread(m_reader);
-        m_writeThread = new Thread(m_writer);
-        m_readThread.setName(m_logName + " Reader");
-        m_writeThread.setName(m_logName + " Writer");
+        m_readThread.setName("Insteon " + m_logName + " Reader");
         m_readThread.start();
+        m_writeThread = new Thread(m_writer);
+        m_writeThread.setName("Insteon " + m_logName + " Writer");
         m_writeThread.start();
```

Comment:
you forgot to call `setDaemon(true)` on the threads

Reply:
done